settings/Control_Port0_Mouse.txt

```diff
@@ -3,81 +3,321 @@
 3DPool
 3DWorldTennis
 A10TankKiller3Disk
+ATrain
+ATrain&ConstructionSet
 AbandonedPlaces
 AbandonedPlaces2
 AdventuresOfRobinHood
-AllNewWorldOfLemmingsAGA
+AlienBreed3D2AGA
 AlienBreed3DAGA
 AlienBreed3DCD32
-AlienBreed3D2AGA
+AllNewWorldOfLemmingsAGA
 Amberstar
 AmberstarFast
-ArcadeSnookerAGA
 ArcadePoolAGA
 ArcadePoolCD32
 ArcadeSnookerAGA
+ArcadeSnookerAGA
 ArcherMacleansPool
 ArmageddonMan
 ArmourGeddon
 ArmourGeddon2
 AshesOfEmpire
-ATrain
-ATrain&ConstructionSet
+B17FlyingFortress
+BAT
+BAT2
+BalanceOfPower
+BalanceOfPower1990Ed
+BanditKings
+Barbarian2Psygnosis
+BarbarianPsygnosis
+BardsTale
+BardsTale2
+BardsTale3
 BardsTaleII
+BargonAttack
+BatmanReturns
 BattleChess
+BattleChess2
 BattleChessCD32
+BattleIsle
+BeneathAStlSky
 BeneathAStlSkyCD32
 BeneathAStlSkyCD32NoVoice
+BillsTomatoGame
+BlackCauldron
+BlackCrypt
+BloodMoney
+Bloodnet
+BloodnetAGA
+Bloodwych
+Bloodwych&ExtendedLevels
```

Comment:
This requires a little thought.... Bloodwych has the option of CD32 controls (as does Mega Lo Mania) ... previously I had allowed the Config Maker to automatically use CD32 controls when it reads the option from the slave header. Might just be  case of making this a host-selected option.